gradle.properties

```diff
@@ -1,5 +1,5 @@
 name=dokan-java
-version=1.1.0
+version=1.2.0
```

Comment:
As long as no new version is released, a `-SNAPSHOT` suffix should be added to the version (if possible).

Reply:
I can make is SNAPSHOT, no prolem. Btw, I saw "1.1.0" somewhere in the java code, hardcoded ...